psi4/src/psi4/libmints/twobody.cc

```diff
@@ -76,10 +76,14 @@ TwoBodyAOInt::TwoBodyAOInt(const IntegralFactory *intsfactory, int deriv)
         screening_type_ = ScreeningType::Schwarz;
     else if (screentype == "CSAM")
         screening_type_ = ScreeningType::CSAM;
+    else if (screentype == "DENSITY")
+        screening_type_ = ScreeningType::Density;
     else
         throw PSIEXCEPTION("Unknown screening type " + screentype + " in TwoBodyAOInt()");
     
     if (screening_threshold_ == 0.0) screening_type_ = ScreeningType::None;
+
+    max_dens_shell_pair_.resize(bs1_->nshell() * bs1_->nshell(), 1.0);
```

Comment:
Why is this necessary?

Reply:
Moved this to the `create_sieve_pair_info()` function, where its other brethren are set up too.